server/src/main/java/org/opensearch/index/translog/NoOpTranslogManager.java

```diff
@@ -93,7 +93,7 @@ public boolean shouldRollTranslogGeneration() {
     public void trimOperationsFromTranslog(long belowTerm, long aboveSeqNo) throws TranslogException {}
 
     @Override
-    public Translog getTranslog(boolean ensureOpen) {
+    public Translog getTranslog() {
```

Comment:
FYI (new change set), the `NoOpTranslogManager` is effectively unusable since every single caller assumes it will never return `null` for `Translog getTranslog()`. Hopefully, `NoOpTranslogManager` will never be used but you may think about introducing `NoOpTranslog`

Reply:
That's correct the `NoOpTranslog` is already being worked upon as a part of #3242.